src/styles/utils/__variables.scss

```diff
@@ -0,0 +1,16 @@
+$month-length:
+  "month-length--28" 28,
+  "month-length--29" 29,
+  "month-length--30" 30,
+  "month-length--31" 31;
+
+$start-day:
+  "start-day--mon" 0,
+  "start-day--tue" 1,
+  "start-day--wed" 2,
+  "start-day--thu" 3,
+  "start-day--fri" 4,
+  "start-day--sat" 5,
+  "start-day--sun" 6;
```

Comment:
You can decrease code length, just use name of the day, example:
mon: 0
tue: 1
etc...